.gitmodules

```diff
@@ -1,3 +1,6 @@
+[submodule "src/openMVG/features/fast"]
+	path = src/openMVG/features/fast
```

Comment:
Would be cool if we keep the Kfast name for the path
Ideally we can think about integrating directly the code, or fork the repo there https://github.com/openMVG-thirdparty in order to never loose the external git repo if a user decide to remove it's account.